extension/content.js

```diff
@@ -282,11 +299,18 @@ $(document).on('click', event => {
 	$target.closest('.js-details-container').toggleClass('refined-github-minimized');
 });
 
+// Handle issue list ajax
+$(document).on('pjax:end', event => {
+	addYoursMenuItem();
```

Comment:
The link has to be different for pulls and issues, thats why theres seperate ifs